### ISSUE TITLE
library path hardcoded?

### ISSUE THREAD

**Issue by ehusd** [DolbyIO/comms-sdk-cpp#1]
Got this error when trying to launch desktop_app:
dyld[92874]: Library not loaded: '/Users/dolbyvoice/builds/dDLzDfbp/1/voxeet/cpp/sdk/ext-lib/macos/ffmpeg/lib/libavcodec.58.dylib'

I set the LD_LIBRARY_PATH pointing to the sdk-release/lib where it does have that library.

**Comment by djova-dolby** [comment]
Yes sorry, this is bug in our installation step which we will fix for beta.2. The path's to ffmpeg libaries are not set to relative locations during the install step on macos.

Until beta.2 is released the we can offer is work around to manually change using _instal_name_tool_. Here are the commands we used, run from the sdk-release directory, to work around, github won't let me upload a script.
```​
install_name_tool -change /Users/dolbyvoice/builds/dDLzDfbp/1/voxeet/cpp/sdk/ext-lib/macos/ffmpeg/lib/libavcodec.58.dylib lib/libavcodec.58.dylib lib/libdolbyio_comms_multimedia_streaming_addon.dylib

install_name_tool -change /Users/dolbyvoice/builds/dDLzDfbp/1/voxeet/cpp/sdk/ext-lib/macos/ffmpeg/lib/libavutil.56.dylib lib/libavutil.56.dylib lib/libdolbyio_comms_multimedia_streaming_addon.dylib

install_name_tool -change /Users/dolbyvoice/builds/dDLzDfbp/1/voxeet/cpp/sdk/ext-lib/macos/ffmpeg/lib/libavformat.58.dylib lib/libavformat.58.dylib lib/libdolbyio_comms_multimedia_streaming_addon.dylib

install_name_tool -change /Users/dolbyvoice/builds/dDLzDfbp/1/voxeet/cpp/sdk/ext-lib/macos/ffmpeg/lib/libavcodec.58.dylib lib/libavcodec.58.dylib bin/desktop_app 

install_name_tool -change /Users/dolbyvoice/builds/dDLzDfbp/1/voxeet/cpp/sdk/ext-lib/macos/ffmpeg/lib/libavutil.56.dylib lib/libavutil.56.dylib bin/desktop_app 

install_name_tool -change /Users/dolbyvoice/builds/dDLzDfbp/1/voxeet/cpp/sdk/ext-lib/macos/ffmpeg/lib/libavformat.58.dylib lib/libavformat.58.dylib bin/desktop_app 
```

**Comment by djova-dolby** [comment]
Fixed in 2.2.0-beta.2 relesae.
